### PR TITLE
add gating question before retirement and social security questions

### PR DESCRIPTION
**Pull request by loumoore**

### CODE REVIEW

File: config/locales/en.yml
Comment (loumoore) [inline config/locales/en.yml:1225]: these are the only new strings. the rest is from running i18n-tasks normalize.

**Review by alaroia-cfa — APPROVED** [review]
Great change! Nice to see stuff become easier for our clients.

Approved.

You'll have to resolve a sorta trivial merge conflict -- it's the schema version number. It should be peaceful to accept whatever is on master for that line.

**Comment by loumoore** [comment]
> You'll have to resolve a sorta trivial merge conflict -- it's the schema version number. It should be peaceful to accept whatever is on master for that line.

@alaroia-cfa I accepted everything in master but also included the line from my change that adds the new column to the intake table. I think that's correct but just to make sure I got that merge right would you mind taking another quick peek before I merge? thanks!

**Comment by alaroia-cfa** [comment]
@loumoore I took the liberty of merging this. Sarah and I checked, and the merge you did was fine.

**Pull request merged**
